test/Geo_Tests/src/Geo_Spec.enso

```diff
@@ -2,6 +2,7 @@ from Standard.Base import all
 from Standard.Table import Table
 
 import Standard.Geo
+
```

Comment:
I see there is some specific style you are using for the imports here.

Do we have it described anywhere? I think if we want to have a consistent style we need some single source of truth describing what it should look like, otherwise when I will be working on my code I will not know what exactly to do to follow the 'right' style. Maybe we can put it as an `.md` file in the `docs/` directory (which generates the dev.enso.org docs).